sqlmesh/core/test/__init__.py

```diff
@@ -20,6 +26,66 @@
     from sqlmesh.core.config.loader import C
 
 
+class ModelTextTestRunner(unittest.TextTestRunner):
```

Comment:
[Nit] We should probably move all of this out of `__init__.py` and into `runner.py`.